plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/util/ResourceUtils.java

```diff
@@ -1120,7 +1120,7 @@ private static String stripDotGit(String uri) {
 	}
 
 	private static String getLastPathSegment(String mangledUri) {
```

Comment:
I believe that this method should be move to org.jboss.tools.openshift.common.core.utils.URIUtils

Reply:
I think that there are several methods that would be best placed in URIUTils such as stripTrailingSlashes, stripDotGit, getProjectNameForURI

I would prefer to keep it for another PR

Reply:
ok, works for me, can we file this move so that we dont forget?